src/main/java/com/vention/stock_market_share/interfaces/AlphaVintageClient.java

```diff
@@ -0,0 +1,16 @@
+package com.vention.stock_market_share.interfaces;
+
+import com.vention.stock_market_share.response.AlphaVintageResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+@Component
+@FeignClient(name = "AlphaVintageClient", url = " ${alphavantage.base.url}")
+public interface AlphaVintageClient {
```

Comment:
https://github.com/Poviraevsergei/StockMarket/blob/master/src/main/java/by/itechart/service/serviceFeign/StockServiceFeign.java

use variant like this, don't use body. just pathVariable here. And move apiKey ti application.property file

Reply:
Done